### ISSUE TITLE
Suggest Old algos MUSTS NOT be used

### ISSUE THREAD

**Issue by LPardue** [httpwg/http-extensions#1929]
```suggestion
A `Representation-Digest` field using insecure hashing algorithms MUST NOT be used in
```

_Originally posted by @martinthomson in https://github.com/httpwg/http-extensions/pull/1919#r799062420_

**Comment by jricher** [comment]
If we end up creating a new registry for the new fields, instead of using the old one, is there a compelling reason to register the old, insecure algorithms in the first place?

**Comment by LPardue** [comment]
See https://github.com/httpwg/http-extensions/issues/1671.

We agreed on that ticket, that there appear to be reasonable use cases that could benefit from Digest fields with algorithms such as MD5 or crc32.

I stand by the outcome of the discussion there. We don't need this document to have an opinion on the algorithms and how they get used. Rather we can state the interop requirement (the algorithm key) and a statement of fact regarding the known state of the protocol (Insecure) based on analysis from other parts of the ecosystem.

**Comment by LPardue** [comment]
To add, I think it's perfectly fine for _something else_ to profile the algorithms that they think are appropriate for a particular usage. Maybe there's guidance we can cpature for such specifications. I just don't think Digest fields should be policing it.

**Comment by ioggstream** [comment]
@LPardue can we close since this is already addressed?

**Comment by ioggstream** [comment]
Closed in #1972
